src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/01 Executors/05 constant-arrival-rate.md

```diff
@@ -78,9 +91,9 @@ The following graph depicts the performance of the [example](#example) script:
 
 Based upon our test scenario inputs and results:
 
-* The desired rate of 30 iterations every 1 second is achieved and maintained for the majority of the test;
-* test scenario runs for the specified 30 second duration;
-* starting with 2 VUs (as specified with `preAllocatedVUs`), k6 automatically adjusts the number of VUs to achieve the desired rate up to the `maxVUs`; for our test, this ended up as 17 VUs;
-* approximately 900 iterations are performed in total, `30s * 30 iters/s`.
+* The desired rate of 30 iterations started every 1 second is achieved and maintained for the majority of the test.
+* The test scenario runs for the specified 30 second duration.
+* Having started with 2 VUs (as specified by the `preAllocatedVUs` option), k6 automatically adjusts the number of VUs to achieve the desired rate, up to the `maxVUs`; for our test, this ended up as 17 VUs.
+* Approximately 900 iterations are performed in total, `30s * 30 iters/s`.
```

Comment:
Exactly 900 should be *started*, approximately seems like it won't do that and it might start more(which it might with the current implementation :thinking: ). But in general the idea is that it will start 900 as long as the resources allow it.

Reply:
Noted 👍🏻